src/components/common/Talk/index.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import withStyles from 'react-jss';
+import cx from 'classnames';
+
+import Button from '../../forms/Button';
+import styles from './styles';
+
+const Talk = props => {
+  const { theme, sheet, classes, className, talk, ...etc } = props;
+  const { image, title, description, id } = talk;
+  return (
+    <article className={cx(classes.root, className)} {...etc}>
+      <div className={cx(classes.main, 'main-transition')}>
+        <div
+          className={classes.image}
+          style={{ backgroundImage: `url(${image})` }}
```

Comment:
You can do this in the styles, but it's ok to leave it here.

Reply:
the goal is to have a full cover image, this is the reason for use inline